yarn-completion.plugin.zsh

```diff
@@ -141,9 +144,6 @@ _yc_zsh_better_yarn_completion() {
 
   # Load custom completion commands
   case "$(_yc_yarn_command)" in
-    add)
```

Comment:
Why did you delete this?

Reply:
Because `_yc_yarn_add_completion ` looks into the `.npm` folder
which is the cache folder for npm but not necessarily for yarn and for yarn --pnp, you cannot you the cache folder anymore (as the name are like `npm-yargs-parser-10.1.0-7202265b89f7e9e9f2e5765e0fe735a905edbaa8`)

Reply:
I tried to fetch the npm repo instead of looking into the cache folder but I spent more than 2 hours trying to make oh-my-zsh work (I don't know how `_values` work)